packages/salesforcedx-vscode-core/package.json

```diff
@@ -803,6 +803,11 @@
           "type": "boolean",
           "default": false,
           "description": "%experimental_deploy_retrieve_description%"
+        },
+        "salesforcedx-vscode-core.experimental.useApexLibrary": {
+          "type": "boolean",
+          "default": true,
+          "description": "%experimental_apex_library_description%"
```

Comment:
We should update the `apex-node` dependency to `0.0.3` so we have access to the latest changes in the library